installation/docker-compose-common.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+
+services:
+  redis-cache:
+    image: redis:latest
+    restart: on-failure
+    volumes:
+      - redis-cache-vol:/data
+
+  redis-queue:
+    image: redis:latest
+    restart: on-failure
+    volumes:
+      - redis-queue-vol:/data
+
+  redis-socketio:
+    image: redis:latest
+    restart: on-failure
+    volumes:
+      - redis-socketio-vol:/data
+
+  mariadb:
+    image: mariadb:10.3
```

Comment:
Is there any particular reason for MariaDB precisely in version 10.3?

Reply:
No particular reason. Just that config changes for anything <=10.2

Used the config from here: https://github.com/frappe/bench/wiki/MariaDB-conf-for-Frappe

Reply:
So I guess it doesn't need to be specified and can be `latest`. If someone will see very precise version may wonder what is the reason for that and not update MariaDB.